framework/wazuh/rbac/orm.py

```diff
@@ -1040,6 +1040,33 @@ def get_users(self) -> Union[list[dict], bool]:
                 user_ids.append(user_dict)
         return user_ids
 
+    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
+                     to_id: Optional[int] = None):
```

Comment:
Please add the docstring

```suggestion
    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
                     to_id: Optional[int] = None) -> None:
```

Reply:
Fixed in 402c4224dd39f3a6dd3cefca2e48f78cd7c872a2.